include/boost/wave/util/cpp_iterator.hpp

```diff
@@ -806,14 +807,12 @@ typename ContextT::position_type pos = act_token.get_position();
 
             // 21 is the max required size for a 64 bit integer represented as a
             // string
-            char buffer[22];
 
-                using namespace std;    // for some systems sprintf is in namespace std
-                sprintf (buffer, "%zd", pos.get_line());
+		std::string buffer = lexical_cast<std::string>(pos.get_line());
```

Comment:
The indentation got messed up somehow.

Reply:
That whole region is strange.... I'll clean it up.